src/views/CreateView.vue

```diff
@@ -348,12 +369,51 @@ export default {
           unit: this.selectedUnit,
           rest: ''
         }
-        this.isFormSubmitted = false
+        this.isExerciseSubmitted = false
         this.areExercisesDisplayed = false
         this.selectedExercise = ''
       } else {
-        this.isFormSubmitted = true
+        this.isExerciseSubmitted = true
       }
+    },
+    async submitWorkout() {
+      if (this.addedExercises.length !== 0) {
+        const { data: { user } } = await supabase.auth.getUser()
+        const { data, error } = await supabase
+          .from('Workouts')
+          .insert([
+            {
+              workout: {
+                workout_name: this.workoutName,
+                exercises: this.addedExercises
+              },
+              user_id: user.id
+            }
+          ])
+          .select()
+    
+        if (data) {
+          this.isInsertionError = false
+          setTimeout(() => {
+            this.$router.push({ name: 'dashboard' })
+          }, 3500)
+        } else if (error) {
+          this.isInsertionError = true
+        }
+      } else {
+        this.isInsertionError = true
```

Comment:
Use early return pattern inside the if. Then you can omit both else if and else
```suggestion
.        return;
        }
      }
        this.isInsertionError = true
```

Reply:
I don't think I can do that because I still need the rest of this function to be executed.